lib/server.js

```diff
@@ -88,7 +88,7 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       // eslint-disable-next-line
       get: callbackify(async (req, res) => {
```

Comment:
We should remove this `linkedDataHandler` and make this is a regular route (this doesn't event accept linked data, so it's not correct). I suspect we don't need `brRest` at all in this package.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/pull/259/commits/b3e22465c61d6dfce42f13416ee92f0936f033fa

the rest of the calls on `brRest` are just `prefers.ld`. Should those stay or be removed along with `bedrock-rest` entirely?

Reply:
Remove bedrock-rest entirely. I don't think the end points are even returning JSON-LD, it's JSON (that *may* have JSON-LD inside it) -- so it's not proper usage anyway.

Reply:
removed `bedrock-rest` here: https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/pull/259/commits/c6139c0e4a9c9cb3ede38ff67e2be8c0121329dc